packages/core/src/common/keybinding.ts

```diff
@@ -128,10 +128,15 @@ export class KeybindingRegistry {
     }
 
     /**
+     * The `checkAvailability` flag with `false` could come handy when we do not want to check whether the command is currently active or not.
+     * For instance, when building the main menu, it could easily happen that the command is not yet active (no active editors and so on)
+     * but still, we have to build the key accelerator.
+     *
      * @param commandId the unique ID of the command for we the associated ke binding are looking for.
+     * @param checkAvailability if `false` then the availability of the command will not be checked. Default is `true`
      */
-    getKeybindingForCommand(commandId: string): Keybinding | undefined {
-        return (this.commands[commandId] || []).find(binding => this.isValid(binding));
+    getKeybindingForCommand(commandId: string, checkAvailability: boolean = true): Keybinding | undefined {
+        return (this.commands[commandId] || []).find(binding => this.isValid(binding, checkAvailability));
```

Comment:
Is it necessary to pass `checkAvailability` flag to `isValid`, could it be like:
```ts
const bindings = this.commands[commandId];
if (!bindings) return undefined;
if (!options.active) return bindings[0];
return bindings.find(binding => this.isActive(binding))
```